packages/widgetbook_core/pubspec.yaml

```diff
@@ -16,10 +16,11 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
+  flutter_lints: ^2.0.0
```

Comment:
We might want to remove this since it's not longer needed

Reply:
Removed 👍